manubot/cite/pubmed.py

```diff
@@ -241,7 +245,7 @@ def get_pmcid_and_pmid_for_doi(doi):
         return {}
     try:
         element_tree = xml.etree.ElementTree.fromstring(response.text)
-    except Exception as error:
+    except Exception:
```

Comment:
In this case, I don't think the exception will be helpful, because it will be a very verbose and unwieldy stacktrace that the XML could not be parsed. We log `response.text`, which will usually contain a non-XML error message that is actually relevant.

Reply:
Sounds good. I am going to approve this PR.